pythonfile/flask_api.py

```diff
@@ -36,29 +39,103 @@ def add_project():
 
     if error == None:
         manifest['id']=uuid.uuid4()
+        #inserts in mongo db
         coll.insert(manifest)
-        elastic.store_json("test", "projects", manifest)
+        #inserts in elasticsearch now
+        res = es.index(index="projects-index", doc_type='Project', id=manifest['id'], body=manifest)
         return make_response(manifest['id'])
     else:
         return make_response((error, '302'))
 
-@app.route('/api/projects/<uuid:project_id>', methods=['GET'])
+@app.route('/api/projects/get/<project_id>')
 def get_project_by_id(project_id):
     res=coll.find_one({'_id':project_id})
     return jsonify(res)
 
-@app.route('/api/projects/<uuid:project_id>', methods=['DELETE'])
+
+@app.route('/api/projects/delete/<project_id>')
 def delete_project(project_id):
-    if coll.delete_one({'_id':project_id}).deleted_count != 0:
+    #deletion from elastic search index
+    try:
+        res = es.delete(index="projects-index", doc_type='Project', id=project_id, refresh=True)
```

Comment:
Please adjust. In case we got a DB inconsistency, we should still try to delete the MongoDB entry. If both fail, we should return an error.